src/Mosaic.tsx

```diff
@@ -108,14 +108,23 @@ export class MosaicWithoutDragDropContext<T extends MosaicKey = string> extends
 
   render() {
     const { className } = this.props;
+    const HTML5WithTouch = {
+      backends: [
+        {
+          backend: HTML5Backend,
+        },
+      ],
+    };
 
     return (
-      <MosaicContext.Provider value={this.childContext as MosaicContext<any>}>
-        <div className={classNames(className, 'mosaic mosaic-drop-target')}>
-          {this.renderTree()}
-          <RootDropTargets />
-        </div>
-      </MosaicContext.Provider>
+      <DndProvider backend={MultiBackend} options={HTML5WithTouch}>
```

Comment:
This should be done in `Mosaic` instead of this class so that it is still possible for consumers to get `MosaicWithoutDragDropContext`